src/ucsi/mod.rs

```diff
@@ -91,14 +92,14 @@ impl TryFrom<u8> for CommandType {
             0x14 => Ok(CommandType::SetPowerLevel),
             0x15 => Ok(CommandType::GetPdMessage),
             0x16 => Ok(CommandType::GetAttentionVdo),
-            0x17 => Ok(CommandType::GetCamCs),
```

Comment:
It looks like these were all contiguous, but now 0x17 doesn't map to anything - is the intent to leave a gap for 0x17? If yes that's fine, but figured I'd double-check in case :)